profiles/tooluse/caliper-tooluse-v1p1.html

```diff
@@ -153,58 +289,398 @@ <h2>Actions</h2>
+      <td>Optional</td>
+      </tr>
+      </tbody>
+      </table>
+      <h4>Example</h4>
+      <pre><code>
+{
+  "id": "urn:uuid:7e10e4f3-a0d8-4430-95bd-783ffae4d918",
+  "type": "AggregateProgress",
+  "value": "12",
+  "valueMax": "25",
+  "metric": "UnitsPassed",
+  "displayName": "Units Passed",
+  "startedAtTime": "2018-08-15T10:15:00.000Z",
+  "endedAtTime": "2018-11-15T10:15:00.000Z"
+}
+      </code></pre>
+    </section>
+        
+        </section>
+        
+        <section id="jsonld">
+          <h2>Representing Caliper as JSON-LD</h2>
 
-    <section class="informative">
-      <h2>Tool Use Profile JSON-LD Examples</h2>
-      <p>Below is an example of a Caliper <code>ToolUseEvent</code> expressed as JSON-LD.  
-        TODO ... </p>
-
-      <figure class="example">
-        <figcaption><code>ToolUseEvent</code> JSON-LD</figcaption>
-        <pre><code>{
-  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/ToolUseProfile-extension",
-  "id": "urn:uuid:cb3878ed-8240-4c6d-9fee-77221810f5e4",
-  "type": "ToolUseEvent",
-  TODO
-}</code></pre>
-      </figure>
+          <p>Individual Caliper events and entity describes are serialized as JSON-LD. IMS Global
+            provides a remote Tool Use Profile JSON-LD <a href=
+            "http://purl.imsglobal.org/ctx/caliper/v1p1/ToolUseProfile-extension">context</a> for mapping
+            <code>ToolUseEvent</code> terms to IRIs. Caliper message producers MUST reference this
+            context in every Caliper 1.1 <code>ToolUseEvent</code> and/or <code>Query</code> describe created
+            and serialized for transmission to a target endpoint.</p>
+
+          <p>See the Caliper 1.1 specification [[!CALIPER-11]] for a discussion of JSON-LD and JSON-LD
+            context handling.</p>
 
-      <p>Below is an example of a ... TODO</p>
+          <figure class="example">
+            <figcaption>JSON-LD Context</figcaption>
+            <pre><code>{
+          "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/ToolUseProfile-extension",
+          . . .
+      }</code></pre>
+          </figure>
 
-      <figure class="example">
-        <figcaption><code>ToolUseResponse</code> describe JSON-LD</figcaption>
-        <pre><code>{
-    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/ToolUseProfile-extension",
-    "id": "urn:uuid:cb3878ed-8240-4c6d-9fee-77221810f5e4",
-    "type": "ToolUseEvent",
-    TODO
+          <section class="informative">
+            <h2>Tool Use Profile JSON-LD Examples</h2>
+            <p>Below is an example of a Caliper <code>ToolUseEvent</code> expressed as JSON-LD.</p>
+
+            <figure class="example">
+              <figcaption><code>ToolUseEvent</code> JSON-LD</figcaption>
+              <pre><code>
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
```

Comment:
I think this context IRI actually needs to be the `v1p1/ToolUseProfile-extension` IRI, otherwise readers of this event will have vocab in it they don't understand.

Reply:
```suggestion
  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/ToolUseProfile-extension",
```

👍 Makes sense until v1p2 ships.

Reply:
Our general intention was that all profile extensions that had been published between core releases would be taken into the core vocabulary with a new core release. Then further extensions for a profile would be based on that new profile, not the older one.